page-templates/stacked.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * Template Name: Stacked
+ *
+ * @package nusa
+ */
+
+get_header();
+?>
+	<main id="main" class="site-main">
+		<section class="section section--fluid">
+			<div class="container--fluid">
```

Comment:
Wondering, should this also have the class `container`? BEM usually has the block level class as well as modifiers.